common/OpTestIPMI.py

```diff
@@ -1075,10 +1076,7 @@ def ipmi_sol_activate(self):
     #
     def ipmi_get_console(self):
         self.ipmi_sol_deactivate()
-        # Waiting for a small time interval as latter versions of ipmi takes a bit of time to deactivate.
-        time.sleep(BMC_CONST.IPMI_SOL_DEACTIVATE_TIME)
         l_con = self.ipmi_sol_activate()
-        time.sleep(BMC_CONST.IPMI_SOL_ACTIVATE_TIME)
         count = 0
```

Comment:
I think  self.ipmi_sol_deactivate() should also be in the below while loop. so that before each time it activates it will deactivate it as ipmi deactivate is not happening properly.